src/SSHDebugPS/RawShell.cs

```diff
@@ -0,0 +1,295 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using liblinux;
+using liblinux.Shell;
+using Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier;
+
+namespace Microsoft.SSHDebugPS
+{
+    public interface IRawShell : IDisposable
```

Comment:
>IRawShell [](start = 21, length = 9)

I find this name very confusing, since many of the actual implementations of this interface don't involve shells.  Maybe `IRawProcess` or `IProcess`?